content/departments/product-engineering/engineering/cloud/devops/managed/upgrade_process.md

```diff
@@ -586,7 +586,8 @@ cd $NEW_DEPLOYMENT/docker-compose && rm docker-compose.yaml && ln -s ../../../go
 ### 5) Sync files to customer instance
 
 ```sh
-go run ../util/cmd/ sync
+# if instance is v1.0, add flag: --v1.0
+go run ../util/cmd/ sync [--v1.0]
```

Comment:
I only wonder if we would like to have `mg` or `go run ../util/cmd/` in handbook documentation?
If mg, I think we should like to this installation setup in this file.

Reply:
I think it's OK to keep `go run` for now until we decide to mark a release